assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -58,17 +63,48 @@ const subHeadingText = {
     monthly or one-time contribution today`,
 };
 
-function contribAttrs(isoCountry: IsoCountry): ContribAttrs {
+function contribCtaText(contribType: Contrib): string {
+  switch (contribType) {
+    case 'RECURRING':
+      return 'Contribute with card or PayPal';
+    default:
+      return 'Contribute with credit/debit card';
+  }
+}
```

Comment:
Since there's no logic here other than the switch, would this work as an object?:

```javascript
const contribCtaText = {
  RECURRING: 'Contribute with card or PayPal',
  ONE_OFF: 'Contribute with credit/debit card',
}
```